### PR TITLE
Php73 upgrade

### PR DESCRIPTION
**Pull request by c-schwarze**
## Description

This branch updates from php5 to php7. It also includes some updates to move to imgix, from cdnX.bethel.edu. There also are some quality of life updates mixed in. 

I'll merge this in on wednesday night when I launch the new server!

## Size and Type of change

- Medium Change - 2+ people need to review this
- New feature

## How Has This Been Tested?

- I tested pretty extensively on the new servers via /etc/hosts updates. It seems to work great, but there might be a few things that fall through the cracks. After this is merged in, i'll swing back to find the remaining issues.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have tested on multiple browsers (Chrome, Firefox, Safari, IE suite)

### CODE REVIEW

File: general-cascade/twig/srcset.html [hunk 1]
```diff
@@ -1,30 +1,30 @@
 {% set rand = random(3) + 1 %}
```
Comment (j-jasik) [inline general-cascade/twig/srcset.html]: Since the "rand" variable was being used to assign pictures to random cdn domains can you remove this completely? Or does it get used somewhere else?

File: events/php/event_feed.php [hunk 2]
```diff
@@ -242,22 +242,24 @@ function inspect_event_page($xml, $categories){
 function display_on_feed_events($page_info){
     global $StartDate;
     global $EndDate;
-    $modifiedStartDate = $StartDate / 1000;
-    $modifiedEndDate = $EndDate / 1000;
 
     //Check if the event falls between the given range.
     if( $StartDate != "" && $EndDate != "" ){
+        $modifiedStartDate = $StartDate / 1000;
```
Comment (j-jasik) [inline events/php/event_feed.php]: Why did you move these variables? Is it more efficient to have them inside the if statements?

File: general-cascade/twig/srcset.html (hunk 1 above)
Comment (c-schwarze) [reply general-cascade/twig/srcset.html]: good catch

File: events/php/event_feed.php (hunk 2 above)
Comment (c-schwarze) [reply events/php/event_feed.php]: I would have preferred them to be on the outside, but this was to fix the php7 warnings. Instead of trying to use $EndDate when it doesn't exist, we check to see if it exists first. So with that in mind, I just put them into the if statements
